docs/cellxgene_census_storage_and_release_policy.md

```diff
@@ -42,7 +42,16 @@ The publication date along with the full URI paths for the `soma` folder and the
 
 * This file MUST be in `json` formats where the parent keys are release identifiers (alias or name).
 * The alias `latest` MUST be present and MUST point to the **Weekly Census release**.
-* The prefix `V` MUST be used followed by an integer counter to label long-term supported Census releases, e.g. `V1`.
+* For a long-term supported ("LTS") release:
+  * A `flags` key MUST be present with a dictionary-typed value, which MUST contain the key `lts` with the value `true`.
+  * An alias MUST exist that points to the release, formatted as `V<#>`, where `<#>` is integer  to label long-term supported Census releases, e.g. `V1`.
```

Comment:
Since we already had the "V<#>" LTS naming policy in place, I stuck with it. Note that we discussed using "LTS-YYYY-MM-DD" in our design meeting. I don't feel strongly either way,

Reply:
Same here!